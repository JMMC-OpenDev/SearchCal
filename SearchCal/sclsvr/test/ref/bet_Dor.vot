version https://git-lfs.github.com/spec/v1
oid sha256:e3873cc74e4b45b54a4c2f90f4ad263975e2aec248e86586b292cb26e18a2e0c
size 153001
