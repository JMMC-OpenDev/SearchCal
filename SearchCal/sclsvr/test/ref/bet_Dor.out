version https://git-lfs.github.com/spec/v1
oid sha256:c014b662f63c52fe5e5613b349fd77fcf17780844bd3b0e0b821f3ab38791217
size 211256
